app/src/app/[lng]/organization/[id]/account-settings/project/index.tsx

```diff
@@ -0,0 +1,851 @@
+                        borderWidth: "1px",
+                        borderStyle: "solid",
+                        borderColor: "content.link",
+                      }}
+                    >
+                      {city.name}
+                    </Tabs.Trigger>
+                  ))}
+                </Tabs.List>
+              </Tabs.Root>
+            )}
+          </AccordionItemContent>
+        </AccordionItem>
+      ))}
+    </AccordionRoot>
+  </Box>
+);
+
+interface ProjectDetailsProps {
+  t: Function;
+  lng: string;
+  router: any;
+  selectedCity: string | null;
+  selectedProjectData: ProjectWithCities | null | undefined;
+  selectedCityData: any; // Define a more specific type if possible
+  organizationName?: string;
+  projectUsers: any[]; // Define a more specific type if possible
+  userList: any[]; // Define a more specific type if possible
+  isLoadingProjectUsers: boolean;
+  tabValue: string;
+  setTabValue: (value: string) => void;
+  setSelectedCity: (value: string | null) => void;
+}
+
+const ProjectDetails: React.FC<ProjectDetailsProps> = ({
+  t,
+  lng,
+  router,
+  selectedCity,
+  selectedProjectData,
+  selectedCityData,
+  organizationName,
+  projectUsers,
+  userList,
+  isLoadingProjectUsers,
+  tabValue,
+  setTabValue,
+  setSelectedCity,
+}) => (
+  <Box w="full">
+    {isLoadingProjectUsers ? (
+      <ProgressLoader />
+    ) : (
+      <Box className="bg-white" p={6} rounded={2} mt={12}>
+        <Flex justifyContent="space-between" alignItems="center" mb={6}>
+          {selectedCity ? (
+            <Box>
+              <BreadcrumbRoot
+                gap="8px"
+                fontFamily="heading"
+                fontWeight="bold"
+                letterSpacing="widest"
+                fontSize="14px"
+                textTransform="uppercase"
+                separator={
+                  <Icon
+                    as={MdChevronRight}
+                    boxSize={4}
+                    color="content.primary"
+                    h="32px"
+                  />
+                }
+              >
+                <BreadcrumbItem>
+                  <BreadcrumbLink
+                    href={`/${lng}/admin`}
+                    color="content.tertiary"
+                    fontWeight="normal"
+                    truncate
+                    className="capitalize"
+                  >
+                    {selectedProjectData?.name}
+                  </BreadcrumbLink>
+                </BreadcrumbItem>
+                <BreadcrumbCurrentLink color="content.link">
+                  <Text truncate lineClamp={1} className="capitalize">
+                    {selectedCityData?.name}
+                  </Text>
+                </BreadcrumbCurrentLink>
+              </BreadcrumbRoot>
+              <Flex mt={2} gap={2}>
+                <CircleFlag
+                  countryCode={
+                    selectedCityData?.countryLocode
+                      ?.substring(0, 2)
```

Comment:
Are you sure that truncating the countryLocode will always give the right results here for the country?